pkg/manager/member/orphan_pods_cleaner.go

```diff
@@ -89,6 +110,23 @@ func (opc *orphanPodsCleaner) Clean(tc *v1alpha1.TidbCluster) (map[string]string
 			return skipReason, err
 		}
 
+		// check api server
+		_, err = opc.kubeCli.CoreV1().PersistentVolumeClaims(ns).Get(pvcName, metav1.GetOptions{})
+		if err == nil {
+			skipReason[podName] = skipReasonOrphanPodsCleanerPVCIsFound
+			continue
+		}
+		if !errors.IsNotFound(err) {
+			return skipReason, err
+		}
+
+		// if the PVC is not found in apiserver (also informer cache) and the
+		// phrase of the Pod is Pending, delete it and let the stateful
+		// controller to create the pod and its PVC(s) again
+		if pod.Status.Phase != v1.PodPending {
+			skipReason[podName] = skipReasonOrphanPodsCleanerPodIsNotPending
+			continue
```

Comment:
Since this is a very lightweight operation, so I think this judgment should be placed at the top to get back quickly.

Reply:
good suggestion, fixed in https://github.com/pingcap/tidb-operator/pull/878/commits/1a352865c2bf31f9747fb7092a4dfc0f3d28eedf.